networks/p2p/dial.go

```diff
@@ -442,6 +443,7 @@ var (
 	errNotWhitelisted     = errors.New("not contained in netrestrict whitelist")
 	errExpired            = errors.New("is expired")
 	errExceedMaxTypedDial = errors.New("exceeded max typed dial")
+	errUpdateDial         = errors.New("Try to update dial candidate with a multichannel peer")
```

Comment:
It's weird the error statement is same with log message.^^;
I suggest some.

```suggestion
	errUpdateDial         = errors.New("is updated to a multichannel peer")
```

Reply:
review reflected